### PR TITLE
docs: add hardware description for portable SecureDrop demo

### PR DESCRIPTION
**Pull request by dachary**
## Status

Ready for review

## Description of Changes

add hardware description for portable SecureDrop demo

## Testing

N/A

## Deployment

N/A

### If you made changes to documentation:

- [x] Doc linting passed locally

### CODE REVIEW

File: docs/development/portable_demo.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Hardware
+--------
+
+* A laptop running the :doc:`staging virtual environment <virtual_environments>`
+* A Tails compatible laptop with a physical radio kill switch
+* Four USB keys prepared for the staging environment running on the laptop
+  * Transfert
+  * Journalist
+  * SVS
+  * Admin
```
Comment (redshiftzero) [inline docs/development/portable_demo.rst]: These nested bullets aren't rendering well:

screen shot 2018-02-12 at 11 12 22 am

Adding a line before `*Transfer` should resolve

File: docs/development/portable_demo.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Portable SecureDrop Demo
+========================
+
+When at a conference or travelling, it is possible to prepare a
```
Comment (redshiftzero) [inline docs/development/portable_demo.rst]: `traveling`

File: docs/development/portable_demo.rst [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+SecureDrop demo using portable hardware and adapted usage scenarios.
+
+Hardware
+--------
+
+* A laptop running the :doc:`staging virtual environment <virtual_environments>`
+* A Tails compatible laptop with a physical radio kill switch
+* Four USB keys prepared for the staging environment running on the laptop
+  * Transfert
```
[the remark this answers is not in the document]
Comment (dachary) [reply docs/development/portable_demo.rst]: done

File: docs/development/portable_demo.rst [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+========================
+
+When at a conference or travelling, it is possible to prepare a
+SecureDrop demo using portable hardware and adapted usage scenarios.
+
+Hardware
+--------
+
+* A laptop running the :doc:`staging virtual environment <virtual_environments>`
+* A Tails compatible laptop with a physical radio kill switch
```
[the remark this answers is not in the document]
Comment (dachary) [reply docs/development/portable_demo.rst]: done

File: docs/development/portable_demo.rst (hunk 1 above)
Comment (dachary) [reply docs/development/portable_demo.rst]: done

**Comment by dachary** [comment]
fixed & rebased & repushed. Thanks for the review. And I will remember to run M-x ispell-buffer to catch trivial spelling errors :-)

**Pull request merged**